configuring/configuring-openshift-builds.adoc

```diff
@@ -1,13 +1,13 @@
-:_content-type: ASSEMBLY
+:_mod-docs-content-type: ASSEMBLY
 [id="configuring-openshift-builds"]
 include::_attributes/common-attributes.adoc[]
-= Configuring {builds-shortname} 
+= Configuring {builds-shortname}
 :context: configuring-openshift-builds
 
 toc::[]
 
 
-A `Build` custom resource (CR) helps you to define the source, build strategy, parameter values, output, retention parameters, and volumes to configure a build. A `Build` resource is available for use within a namespace. 
+A `Build` custom resource (CR) helps you to define the source, build strategy, parameter values, output, retention parameters, and volumes to configure a build. A `Build` resource is available for use within a namespace.
```

Comment:
"helps you" is a form of anthrophomorphism. How about:

By using/creating a `Build` custom resource (CR), you can define the source, build strategy, parameter values, output, retention parameters, and volumes to configure a build. You can use a `Build` CR in a cluster's namespace.

Reply:
I have changed it in this manner: In a `Build` custom resource (CR), you can define the source, build strategy, parameter values, output, retention parameters, and volumes to configure a build.

I think the second sentence is okay, we can leave it as is.